TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/EWCWalesImport/InductionImporter.cs

```diff
@@ -0,0 +1,452 @@
+using System.Globalization;
+using System.Text;
+using CsvHelper;
+using Microsoft.Extensions.Logging;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Query;
+using TeachingRecordSystem.Core.Dqt;
+using TeachingRecordSystem.Core.Dqt.Queries;
+
+namespace TeachingRecordSystem.Core.Jobs.EwcWalesImport;
+
+public class InductionImporter(ICrmQueryDispatcher crmQueryDispatcher, ILogger<InductionImporter> logger)
+{
+    public async Task<(int TotalCount, int SuccessCount, int DuplicateCount, int FailureCount, string FailureMessage)> ImportAsync(StreamReader csvReaderStream, Guid IntegrationTransactionId, string fileName)
```

Comment:
I would change this tuple into its own type, to reduce the verbosity a little:
```cs
public record InductionImportResult(int TotalCount, int SuccessCount, int DuplicateCount, int FailureCount, string FailureMessage);
```

```suggestion
    public async Task<InductionImportResult> ImportAsync(StreamReader csvReaderStream, Guid integrationTransactionId, string fileName)
```